pkg/controllers/workapplier/controller.go

```diff
@@ -352,17 +352,34 @@ const (
 )
 
 type manifestProcessingBundle struct {
-	manifest                *fleetv1beta1.Manifest
-	id                      *fleetv1beta1.WorkResourceIdentifier
-	manifestObj             *unstructured.Unstructured
-	inMemberClusterObj      *unstructured.Unstructured
-	gvr                     *schema.GroupVersionResource
+	// The manifest data in the raw form (not decoded yet).
+	manifest *fleetv1beta1.Manifest
+	// The work resource identifier of the manifest.
+	// If the manifest data cannot be decoded as a Kubernetes API object at all, the identifier
+	// will feature only the ordinal of the manifest data (its rank in the list of the resources).
+	// If the manifest data can be decoded as a Kubernetes API object, but the API is not available
+	// on the member cluster, the resource field of the identifier will be empty.
+	id *fleetv1beta1.WorkResourceIdentifier
+	// A string representation of the work resource identifier (sans the resources field).
+	// This is only populated if the manifest data can be successfully decoded.
+	wriStr string
```

Comment:
```suggestion
	workResourceIdentifierStr string
```
prefer to use the full name as wri is not a common initialism.

Could you please add a comment about the string format or an example?

Reply:
Yeah, will do 👌